### DISCUSSION TITLE
Does gorilla provide command description ?

### DISCUSSION THREAD

**Discussion by Azzedde** [ShishirPatil/gorilla#124] [Q&A]
Hello,
I'm interested in knowing more details about the other commands (except the first one) that gorilla suggest, are they provided on the response ? if yes how can I access them ?

**Comment by ShishirPatil** [comment]
Hey @Azzedde yes indeed! We are constantly changing the recipe to make sure the tool is usable. So the other commands are usually Gorilla with a different temperature, or a fraction of the times, commands generated by GPT/Claude. This additionally also helps us benchmark how well Gorilla performs via-a-vis close-sourced models. Let me know if you have any other questions!!
